packages/flutter/lib/src/rendering/error.dart

```diff
@@ -9,6 +9,7 @@ import 'object.dart';
 
 const double _kMaxWidth = 100000.0;
 const double _kMaxHeight = 100000.0;
+const String line = '\n──────────────────────\n\n';
```

Comment:
Add a comment to that effect. Also, have you checked what it looks like when the exception happens inside a 100x100 widget? Or a 10x200 widget, or a 200x10 widget? :-)